File: QuanLyHienMauApi/QuanLyHienMauApi/Models/XacNhanDoiTac.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace QuanLyHienMauApi.Models
{
    public class XacNhanDoiTac
    {
        [Key]
        public int ID { get; set; }

        public int YeuCauID { get; set; }

        public int BenhVienChiaSeID { get; set; }

        public DateTime NgayXacNhan { get; set; } = DateTime.Now;

        public int TrangThai { get; set; } = 0; 

       
        [ForeignKey("YeuCauID")]
        public virtual YeuCauMau YeuCau { get; set; }

        [ForeignKey("BenhVienChiaSeID")]
        public virtual NguoiDung BenhVienChiaSe { get; set; }
    }
}

File: HienMau/app/src/main/java/com/example/hienmau/models/XacNhanDoiTac.java
package com.example.hienmau.models;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class XacNhanDoiTac implements Serializable {
    @SerializedName("id")
    public int id;

    @SerializedName("yeuCauID")
    public int yeuCauID;

    @SerializedName("benhVienChiaSeID")
    public int benhVienChiaSeID;

    @SerializedName("ngayXacNhan")
    public String ngayXacNhan;

    @SerializedName("trangThai")
    public int trangThai;

    @SerializedName("yeuCau")
    public YeuCauMau yeuCau;

    @SerializedName("benhVienChiaSe")
    public NguoiDung benhVienChiaSe;

    // Thuộc tính ảo phục vụ bọc dữ liệu hiển thị giống Tình nguyện viên
    @SerializedName("nguoiHien")
    public NguoiDung nguoiHien;

    @SerializedName("trangThaiConfirm")
    public int trangThaiConfirm;
}
